_el/exercises.md

```diff
@@ -1,169 +1,95 @@
-> 1. function call, then assignment and subtraction simultaneously
-> 1. none of the above
-
-The second classic type of programming exercise is *code and run*
-(C&R), in which the learner writes code that produces a specified
-output. C&R exercises can be as simple or as complex as the teacher
-wants, but for in-class use, they should be brief and have only one or
-two plausible correct answers. For novices, it's often enough to ask
-them to call a specific function: experienced teachers often forget how
-hard it can be to figure out which parameters go where. For more
-advanced learners, figuring out which function to call is more engaging
-and a better gauge of their understanding.
-
-> **Code & Run**
-> 
-> The variable `picture` contains a full-color image read from a file.
-> Using one function, create a black and white version of the image and
-> assign it to a new variable called `monochrome`.
+Κάθε καλός ξυλουργός έχει μια σειρά από κατσαβίδια και κάθε καλός δάσκαλος έχει διαφορετικά είδη ασκήσεων διαμορφωτικής αξιολόγησης τα οποία μπορεί να χρησιμοποιήσει για να ελέγχει τι μαθαίνουν πραγματικά οι μαθητές, να τους βοηθά να εξασκούν τις νέες τους δεξιότητες, και να διατηρεί ζωντανό το ενδιαφέρον τους. Αυτό το κεφάλαιο αρχίζει περιγράφοντας διάφορα είδη ασκήσεων που μπορείτε να χρησιμοποιήσετε για να ελέγξετε αν η διδασκαλία σας ήταν αποτελεσματική. Στη συνέχεια κάνει μια επισκόπηση της τρέχουσας κατάστασης στην τεχνολογία αυτοματοποιημένη αξιολόγησης και ολοκληρώνεται με μια διερεύνηση των συζητήσεων, έργων, και άλλων σημαντικών ειδών εργασίας τα οποία απαιτούν περισσότερη ανθρώπινη προσοχή για να αξιολογηθούν. Η συζήτησή μας βασίζεται εν μέρει στην [Τράπεζα Ερωτήσεων του Canterbury] (http://web-cat.org/questionbank/)[[Sand2013] (# CITE)], η οποία περιέχει καταχωρήσεις για διάφορες γλώσσες προγραμματισμού και θέματα που αφορούν την εισαγωγικού επιπέδου υπολογιστική.
 
-Write and run exercises can be combined with MCQs. For example, this MCQ
-can only be answered by running the Unix `ls` command:
+## Οι κλασσικές {#s: ασκήσεις-κλασικά}
 
-> **Combining MCQ with Code & Run**
-> 
-> You are in the directory `/home/greg`. Which of the following files is
-> *not* in that directory?
-> 
+Όπως συζητήθηκε στην Ενότητα [s:models-formative-assessment](#SECTION), οι *ερωτήσεις πολλαπλών επιλογών* (ΕΠΕ) είναι πιο αποτελεσματικές όταν οι λανθασμένες απαντήσεις βοηθούν στη διερεύνηση συγκεκριμένων παρανοήσεων. Σύμφωνα με την ταξινόμηση του Bloom ([s:process-objectives](#SECTION)), οι ΕΠΕ συνήθως σχεδιάζονται για να ελέγξουν την ανάκληση και κατανόηση ενός θέματος ("Ποια είναι η πρωτεύουσα του νομού Πέλλας;"), αλλά μπορούν επίσης να απαιτούν από τους εκπαιδευόμενους να ασκούν τις κριτικές τους ικανότητες.
+
+> ** Μια ερώτηση πολλαπλής επιλογής **
+>
+> Με ποιά σειρά γίνονται οι πράξεις όταν ο υπολογιστής αξιολογεί την έκφραση `price = addTaxes(cost - discount)`;
+>
+> 1. αφαίρεση, κλήση συνάρτησης, αντιστοίχιση
```

Comment:
αντιστοίχιση--> ανάθεση

Reply:
Thanks @pdelias ! Fixed in a new commit.